packages/@react-spectrum/s2/stories/SelectBoxGroup.stories.tsx

```diff
@@ -312,5 +307,6 @@ export const AllSlotCombinations: Story = {
       </div>
 
     </div>
-  )
+  ),
+  tags: ['!autodocs']
```

Comment:
we could either remove this and rely on chromatic or leave this in storybook and remove it from the docs